src/lib/storage/column_iterables.hpp

```diff
@@ -80,6 +80,49 @@ class ColumnIterable {
     });
   }
 
+  /**
+   * @defgroup Function for the materialization of values and nulls
+   * @{
+   */
+
+  /**
+   * Materialize all values in this iterable.
+   * Fallback implementation. Derived iterables can implement their own, faster version.
```

Comment:
Could you add this line to the group instead? 

`Fallback implementation. Derived iterables can implement their own, faster version.`

Perhaps:
`The following implementations may be overridden by derived classes`

Reply:
done